test/test_mutant_epitope_predictor.py

```diff
@@ -21,6 +21,9 @@
 from topiary import MutantEpitopePredictor, epitopes_to_dataframe
 from varcode import Variant, VariantCollection
 
+ensembl.download()
```

Comment:
I had that as the first commit, but felt that logic should be coupled with where it's used. When coupled with the test using it, no one needs to "remember" to add some random script.